pkg/util/util_test.go

```diff
@@ -18,6 +22,54 @@ func TestToPtr(t *testing.T) {
 	require.Equal(t, &stringVar, ToPtr(stringVar))
 }
 
+func TestToInt32Ptr(t *testing.T) {
+	var int32Var = int32(2)
+	require.Equal(t, &int32Var, Int32Ptr(int32Var))
+}
+
+func TestToInt64Ptr(t *testing.T) {
+	var int64Var = int64(2)
+	require.Equal(t, &int64Var, Int64Ptr(int64Var))
+}
+
+func TestUseServerSideApply(t *testing.T) {
+	require.Equal(t, patchType, client.Merge)
+	UseServerSideApply()
+	require.Equal(t, patchType, client.Apply)
+
+	// reset
+	patchType = client.Merge
```

Comment:
this is going to mess with our other tests. There's a reason we leave it as it was. Don't need to test UseServerSideApply()

Reply:
removed